poll/poll.py

```diff
@@ -508,6 +511,7 @@ def tally_detail(self):
                 answer['choice'] = True
             try:
                 answer['percent'] = round(answer['count'] / float(total) * 100)
+                answer['leader'] = answer['count'] == highest_count
```

Comment:
Shouldn't this be outside the `try/except` to keep the block explicitly catching the lines where that error might happen?